src/main/java/org/candlepin/model/ConsumerContentOverrideCurator.java

```diff
@@ -31,4 +33,12 @@ public ConsumerContentOverrideCurator() {
     protected ConsumerContentOverride createOverride() {
         return new ConsumerContentOverride();
     }
+
+    public Collection<ConsumerContentOverride> listConsumerContentOverrides(Consumer consumer) {
```

Comment:
Where will this be used? This doesn't have a corresponding usage from a resource method.

Reply:
sorry I forgot to remove it I thought we are calling the database in the resource layer so I wanted to extract it into the curator, but then I noticed the changed getList() and how it work.